spec/controllers/doorkeeper/authorizations_controller_spec.rb

```diff
@@ -284,6 +284,35 @@ def expect_successful_callback!
       end
     end
 
+    context 'with multiple prompt values' do
+      it 'when select_account+login' do
+        authorize! prompt: 'select_account login'
+        expect(response).to redirect_to('/select_account')
+      end
+
+      it 'when login+select_account' do
+        authorize! prompt: 'login select_account'
+        expect(response).to redirect_to('/select_account')
+      end
+
+      it 'when consent+select_account' do
+        authorize! prompt: 'consent select_account'
+        expect(response).to redirect_to('/select_account')
+      end
+
+      it 'when select_account+consent' do
+        authorize! prompt: 'select_account consent'
+        expect(response).to redirect_to('/select_account')
+      end
+
+      # FIXME:
+      it 'when login+consent' do
+        expect do
+          authorize! prompt: 'login consent'
+        end.to raise_error AbstractController::DoubleRenderError
+      end
```

Comment:
this is existing bug, but not trying to fix it in this pull request. just making the fact visible.

Reply:
tried to fix, and seems succeeded?